static/app/views/insights/common/queries/useSpanSamples.tsx

```diff
@@ -132,7 +132,7 @@ export const useSpanSamples = <Fields extends NonDefaultSpanSampleFields[]>(
           firstBound: max * (1 / 3),
           secondBound: max * (2 / 3),
           upperBound: max,
-          additionalFields,
+          additionalFields: [SpanIndexedField.ID, ...additionalFields],
```

Comment:
We're already returning `span_id` from `spans-samples`, so the id field shouldn't be needed. (id = span_id). 

Looking through your code, its only used to pass in the correct sample into the event handlers like `onClickSample(spanSamplesById[sample.id])`
But `spanSamplesById` is generated from `span_id` anyways!



If ID is preferred however, we should just return that from the endpoint imo!